src/actionTextFileValidations.js

```diff
@@ -0,0 +1,73 @@
+export class ActionTextFileValidations {
+    // Make sure to update VALID_ATTACHMENT_TYPES in rad_common if you add or remove any file types
```

Comment:
FYI we thought about doing an endpoint to do the validations on the rad_common side, so we don't have to put these file types hardcoded, but there are issues using an async function and the trix event. Probably a few other ways to do it, but thought for now it might be ok to hardcode these since these don't change often